.idea/workspace.xml

```diff
@@ -3,7 +3,6 @@
   <component name="ChangeListManager">
```

Comment:
We don't want workspace.xml checked in. This is a per user file.

Reply:
This just slipped in there. I'll try to be more cognizant.